### PR TITLE
Created website basement

### PR DESCRIPTION
**Pull request by ArkanKovanAbdullah**
## created website basement

- Html base added
- css base added
- head title added

### CODE REVIEW

**Comment by ArkanKovanAbdullah** [comment]
# ♻️♻️♻️  Status: Changes Required ♻️♻️♻️ 
d9fcf4f4995e18ae4fb9c81fe55832dc 
hi @ArkanKovanAbdullah 

Your project looks good. You are almost done. However, there are some issues you should fix and some improvements which can help. Please fix them and your project will be approved.

## 💯 Highlights 💯

- The right github flow
- Inputs have the correct types
- HTML basement works
- CSS file is connected in the right manner
- background picture aligned correctly

## ♻️ Request changes ♻️

1.  ***No linter added*** Please add linters and make sure there are no errors. linters are useful to keep your code clean and understandable. Here is a video on: [how you can add linters to your github repository.](https://www.youtube.com/watch?v=fL8de_m8imY) 
2.  ***Container width*** Your container width is fixed i suggest you chnage that and make it fill the width of the content in it. 
28 09 2022_23 58 49_REC
3. ***Word spacing*** Please give this text the correct spacing. These small things really effect how you website looks. 
29 09 2022_00 08 45_REC
4. ***margin and padding*** I suggest you set the margin and padding of all of the elements to zero and then work on margin and padding of elements individually. This can be done by: 
29 09 2022_00 15 42_REC
6. ***Checkboxes Alignment*** Please make the all checkboxes show in a row. You can do this by making a parent containg and making a flexbox and make the flex-direction to row. 
7. ***form content Alignment*** Please make the contents in the form center horizontally.

##

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

# **Cheers and Happy coding! 💻 👏**

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @ArkanKovanAbdullah , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- Polite Comments. ✔


## Required Changes ♻️

- [x] You did a good job reviewing this project. 👍️ But I would recommend when you want to request a change to use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request), instead of taking screenshots of the code. Take a look at [this video](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d) to see how to use the inline comment feature.

- [ ] Why is it important to use inline comments when reviewing a code? 
 It helps the pull request owner to know what is going wrong and where he should implement the change.😊

- [ ] User screenshots when you want to compare two things to show the author of the project how his result should look like.
- [ ] Remember to explain `why` when you request a change because it is very useful to the owner of the pull request. I saw you did it in your first comment. That's great 💪.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <input class= "block" type="radio" value="Bab" name="group1">
+        <input class= "block" type="radio" value="Nan" name ="group2">
+        <input  class= "block" type="radio" value="Bab" name="group2">
+
+        <input class= "block" type="checkbox" value="i dont know">
+        <input class= "block" type="checkbox" value="i do know">
+        <textarea  class= "block" rows="4" cols="50">
+        </textarea>
+        <button class="block" type="submit" id="submit">Submit</button>
+    </form>
```
Comment (ArkanKovanAbdullah) [inline index.html:37]: - [ ] Please make the contests inside this form center horizontally. This makes the website look more alike the template.

**Review by ArkanKovanAbdullah — COMMENTED** [review]
# ♻️♻️♻️  Status: Changes Required ♻️♻️♻️ 
9ac08e6b84e39c5dc68224c3c37a9514

hi @ArkanKovanAbdullah 

Your project looks good. You are almost done. However, there are some issues you should fix and some improvements which can help. Please fix them and your project will be approved.

## 💯 Highlights 💯

- The right GitHub flow
- Inputs have the correct types
- HTML basement works
- CSS file is connected in the right manner
- background picture aligned correctly

## ♻️ Requested changes ♻️

1.  ***No linter added*** Please add linters and make sure there are no errors. linters are useful to keep your code clean and understandable. Here is a video on: [how you can add linters to your GitHub repository.](https://www.youtube.com/watch?v=fL8de_m8imY) 
2.  ***Container width*** Your container width is fixed i suggest you change that and make it fill the width of the content in it. also make the elements inside it take at least 90% of their parent, So that it resembles the template. 
Yours:
28 09 2022_23 58 49_REC

Template:
29 09 2022_01 55 31_REC

3. ***margin and padding*** I suggest you set the margin and padding of all of the elements to zero and then work on margin and padding of elements individually. This can be done by: 
29 09 2022_00 15 42_REC
4. ***Checkboxes Alignment*** Please make the all checkboxes show in a row. You can do this by making a parent container and make a flexbox and make the flex-direction to row. 
5. ***form content Alignment*** Please make the contents in the form center horizontally.
6. ***Descriptive read me file*** Please add a descriptive read me file.
7. ***Pull request title and description*** Please give your pull request a clear title, and a more descriptive description. 
8. ****check the comments under the review ⬇️****
#

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

d9fcf4f4995e18ae4fb9c81fe55832dc 
# **Cheers and Happy coding! 💻 👏**

**Review by franclobo — APPROVED** [review]
Hi @ArkanKovanAbdullah, 

# STATUS: APPROVED 😃 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Correct use of resume ✔️
- Good use of inline comments ✅
- Comments are clear and easy to understand ☑️
- Comments are polite 💯
- Comments include an explanation 💯

## Optional suggestions

- [ ] **[OPTIONAL]** kindly use emojis in the inline comments also to make your code review more human-friendly 👽.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**